xpk.py

```diff
@@ -5093,6 +5128,22 @@ def directory_path_type(value):
           'to parse jobs that match the pattern or provide a job name to view a single job.',
     required=False,
 )
+
+workload_list_parser.add_argument(
+    '--wait-for-job-completion',
+    type=str,
+    default=None,
+    help='The name of the job to wait on.',
+    required=False,
+)
+
+workload_list_parser.add_argument(
+    '--timeout',
+    type=int,
+    default=None,
+    help='Amount of time to wait for job in seconds.',
```

Comment:
Nit: Can you add to the help details what the default behavior is? (seems like timeout is set to infinite) and also can you add a similar comment to the readme?

Reply:
Done! If no timeout is specified then it is set to the max timeout (1 week)